modules/core/src/controllers/map-controller.js

```diff
@@ -22,7 +22,7 @@ function clamp(value, min, max) {
   return value < min ? min : value > max ? max : value;
 }
 
-class MapState extends ViewState {
+export class MapState extends ViewState {
```

Comment:
This is exported for testing

Reply:
This may not be obvious to the reader. Just add a comment, or perhaps create a TEST_EXPORTS symbol?

```
export const TEST_EXPORTS = {MapState};
```